### DISCUSSION TITLE
What Is the qoutes directory?

### DISCUSSION THREAD

**Discussion by Shluffy** [theforumhelpers/theforumhelpers.github.io#222] [Q&A] [answered]
^

**Comment by Accio1** [comment, marked as answer]
https://github.com/theforumhelpers/theforumhelpers.github.io/discussions/10

**Reply by Shluffy** [reply]
@Accio1 are we making it? because if not we should probably remove this

**Reply by BarelySmooth** [reply]
I think it would be a good idea. The website is currently limited in functionality - this might increase the use of the website as long as it is _properly_ maintained on a regular basis

**Reply by Shluffy** [reply]
> I think it would be a good idea. The website is currently limited in functionality - this might increase the use of the website as long as it is _properly_ maintained on a regular basis

quickreply is hopefully getting redone soon, and we might want to only do one at a time. Although we could make it now
